python/arcticdb/version_store/library.py

```diff
@@ -1310,9 +1310,9 @@ def finalize_staged_data(
         symbol : `str`
             Symbol to finalize data for.
 
-        mode : `StagedDataFinalizeMethod`, default=StagedDataFinalizeMethod.WRITE
-            Finalize mode. Valid options are WRITE or APPEND. Write collects the staged data and writes them to a
-            new version. Append collects the staged data and appends them to the latest version.
+        mode : Union[`StagedDataFinalizeMethod`, str], default=StagedDataFinalizeMethod.WRITE
```

Comment:
I think lowercase `write, append` will match both Pandas and our `QueryBuilder` APIs better.